src/components/views/rooms/MemberInfo.js

```diff
@@ -440,20 +440,35 @@ module.exports = withMatrixClient(React.createClass({
         const roomId = this.props.member.roomId;
         const target = this.props.member.userId;
         const room = this.props.matrixClient.getRoom(roomId);
-        const self = this;
-        if (!room) {
-            return;
-        }
-        const powerLevelEvent = room.currentState.getStateEvents(
-            "m.room.power_levels", "",
-        );
-        if (!powerLevelEvent) {
-            return;
-        }
+        if (!room) return;
+
+        const powerLevelEvent = room.currentState.getStateEvents("m.room.power_levels", "");
+        if (!powerLevelEvent) return;
+
         if (powerLevelEvent.getContent().users) {
-            const myPower = powerLevelEvent.getContent().users[this.props.matrixClient.credentials.userId];
+            const myUserId = this.props.matrixClient.getUserId();
+            const QuestionDialog = sdk.getComponent("dialogs.QuestionDialog");
+
+            if (myUserId === target) {
```

Comment:
I think we should explicitly check that the power level is decreasing. There should also be a comment explaining that this is worth warning about because (if the change is successful) it implies that the new power level is lower than that required to change power levels.

Reply:
@lukebarnard1 well you can't raise your own PL even if your PL is high enough to change PLs, so it is irreversible in all cases, you can *only* decrease your own PL.

Reply:
Okay, fair enough just make sure to comment about it :+1: